src/Build/Logging/FancyLogger/FancyLogger.cs

```diff
@@ -10,14 +10,10 @@ namespace Microsoft.Build.Logging.FancyLogger
     public class FancyLogger : ILogger
     {   
         private Dictionary<int, FancyLoggerProjectNode> projects = new Dictionary<int, FancyLoggerProjectNode>();
-
         private bool Succeeded;
-
-        private float existingTasks = 1;
-        private float completedTasks = 0;
-
         public string Parameters {  get; set; }
-
+        public float StartedProjects = 0;
+        public float FinishedProjects = 0;
```

Comment:
```suggestion
        public int StartedProjects = 0;
        public int FinishedProjects = 0;
```

Or maybe `uint`?

Reply:
Oh, I see, this is for division to get the progress number later? I think I'd rather see a cast there, but you could just comment this.